api/python/provisioner/commands/__init__.py

```diff
@@ -885,6 +906,17 @@ def run(self, targets):  # noqa: C901 FIXME
                     # fail to start - fail gracefully:  disable
                     # maintenance in the background
                     cluster_maintenance_disable(background=True)
+                elif isinstance(
```

Comment:
that block is not going to occur since these exception won't get out of [that](https://github.com/Seagate/cortx-prvsnr/blob/73ebb989586a3a1d84dd19c8b47ba2fe3b71eadf/api/python/provisioner/commands/__init__.py#L820-L845) scope